modules/veteran/spec/requests/v0/representatives_spec.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'VSO representative spec', type: :request do
+  let(:representative) { FactoryBot.create(:representative, first_name: 'Bob', last_name: 'Smith', poa: '1BQ') }
+
+  before do
+    representative
```

Comment:
You don't have to do this, you can do `let!` and it'll run it rather than lazy loading it

Reply:
This has been updated